src/components/updatemenus/draw.js

```diff
@@ -181,8 +191,8 @@ function drawHeader(gd, gHeader, gButton, menuOpts) {
         .text('▼');
```

Comment:
Maybe we could add some logic for that `▼` character around `orientation` and `openreverse`.

For example, `orientation: 'h'` / `openreverse: true` would put some :arrow_backward: character to the left of the header text. `orientation: 'v'` / `openreverse: ture` would put some :arrow_up: character etc.